.nvmrc

```diff
@@ -0,0 +1 @@
+10.16
```

Comment:
Whats `10.16` for?

Reply:
It becomes something that needs to be maintained and something that will eventually go stale. If its absolutely necessary It's okay, but this is not necessary.

Reply:
Well, that's fair enough, I'll remove it. 👍